### DISCUSSION TITLE
v0.11.0-preview2

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#907] [Announcements]
https://github.com/Blinue/Magpie/compare/v0.11.0-preview1...v0.11.0-preview2

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.11.0-preview2'>v0.11.0-preview2</a>.</em>

**Comment by silkbad** [comment]
For touch support:  Can you add an option "Emulate Mouse Click" ?
Reason:  Magpie enables use of old games in full-screen, but if those games were not designed for touch input, certain buttons are still un-clickable with Touch (even without Magpie). 
A million thanks for this amazing program and your development :)

**Reply by Blinue** [reply]
Magpie only offers scaling functionality, and compatibility issues beyond that are beyond its scope. (As far as I know, Windows typically simulates touch input as mouse input by default unless a window explicitly declares touch support.)
